test/recipes/checkhandshake.pm

```diff
@@ -0,0 +1,128 @@
+#! /usr/bin/env perl
```

Comment:
No...thanks (it would be really nice if perl actually gave me useful error messages to detect problems like that)...but now I get lots of lines like:
````
../test/recipes/70-test_tls13messages.t .. Bareword "recipes::checkhandshake::ALL_HANDSHAKES" not allowed while "strict subs" in use at ../test/recipes/70-test_tls13messages.t line 35.
Bareword "recipes::checkhandshake::ALL_HANDSHAKES" not allowed while "strict subs" in use at ../test/recipes/70-test_tls13messages.t line 35.
Bareword "recipes::checkhandshake::ALL_HANDSHAKES" not allowed while "strict subs" in use at ../test/recipes/70-test_tls13messages.t line 35.
````